session1/solutions/assignment3/Rakefile

```diff
@@ -1 +1,20 @@
-# assignment 3 solution
+namespace :files do
+  desc 'rename file extention yaml to yml '
+  task :rename_yaml do
+    original_extention = ".yaml"
+    new_extention = ".yml"
+    file = ENV["FILE"]
+    file_basename = File.basename(file,original_extention)
+    directory = File.dirname(file)
+    old_file_name = directory + "/" + file_basename + original_extention
+    new_file_name = directory + "/" + file_basename + new_extention
+  
+    if File.exist?(file) == false then 
+      puts "== files:rename_yaml == Error: File #{file} doesn't exist"
```

Comment:
The correct way of handling such cases is to raise an exception:
```ruby
raise "File #{file} doesn't exists"
```

This would also exit the program so `exit` not needed.